app/views/request/_request_sent.html.erb

```diff
@@ -1,19 +1,66 @@
-          :law_used_full => @info_request.law_used_full) %>
-  </p>
+<div id="content">
+  <div class="request-sent-message" id="notice">
+    <h1>
+      <%= _("Your {{law_used_full}} request has been sent",
+            :law_used_full => @info_request.law_used_full) %>
+    </h1>
+    <div class="request-sent-message__row">
+      <div class="request-sent-message__column-1">
+        <p class="subtitle">
+          <%= _("<strong>We will email you</strong> when there is a response, or after " \
+                "{{late_number_of_days}} working days if the authority still hasn't " \
+                "replied by then.",
+                :late_number_of_days => AlaveteliConfiguration.reply_late_after_days) %>
+        </p>
 
-  <p>
-    <%= _("<strong>We will email you</strong> when there is a response, or after " \
-          "{{late_number_of_days}} working days if the authority still hasn't " \
-          "replied by then.",
-          :late_number_of_days => AlaveteliConfiguration.reply_late_after_days) %>
-  </p>
+        <h2><%= _("Share your request") %></h2>
 
-  <p>
-    <%= _("If you write about this request (for example in a forum or a blog) " \
-          "please link to this page, and add an annotation below telling people " \
-          "about your writing.") %>
-  </p>
+           <%= link_to image_tag("next-step-twitter.png",
+                                 :alt => _("Tweet it"),
+                                 :width => "120",
+                                 :height => "37"),
+                       "https://twitter.com/intent/tweet?" << {
+                :url => request.url,
+                :via => AlaveteliConfiguration.twitter_username,
+                :text => "'#{ @info_request.title }'",
+                :related => _('alaveteli_foi:The software that runs {{site_name}}', :site_name => site_name)
+             }.to_query %>
+
+           <%= link_to image_tag("next-step-facebook.png",
+                                 :alt => _("Share on Facebook"),
+                                 :width => "120",
+                                 :height => "37"),
+                      "https://www.facebook.com/sharer/sharer.php?" << {
+               :u => request.url
+             }.to_query %>
+
+        <h2><%= _("Keep your request up to date") %></h2>
+        <p>
+          <%= _('If you write about this request ' \
+                '(for example in a forum or a blog) ' \
+                'please link to this page, and <a href="{{url}}">add an ' \
+                'annotation</a> below telling people ' \
+                'about your writing.', :url => new_comment_url(:url_title => @info_request.url_title).html_safe) %>
+        </p>
+      </div>
+      <div class="request-sent-message__column-2">
+        <div class="what-next">
+          <h2><%= _("What next?") %></h2>
+          <ul class="what-next__list">
+            <li>
+              <a href="#"><%= _("Change how you'd like to be notified of changes " \
```

Comment:
Fixed in https://github.com/mysociety/alaveteli/commit/cd1786ab79df6e1e9f23a2482c2d64ded872aa1b